tvb_documentation/tutorials/tutorial_0_GettingStarted.rst

```diff
@@ -29,8 +28,7 @@ other users and/or link data to other TVB projects.  After this tutorial you
 should be able to import a new dataset, export a project and share it with a colleague.
 
 For your convenience, all the data were already generated. You can download
-the LinkandShare_a.zip and LinkandShare_b.zip files in the `TVB sharing area
-<http://www.thevirtualbrain.org/tvb/zwei/client-area/public>`_. We’ll only go
+the LinkandShare_a.zip and LinkandShare_b.zip files in the `TVB dataset repo <https://zenodo.org/record/4263723>`_. We’ll only go
```

Comment:
I think these projects are missing now... or?

Reply:
All these previous link pointed to a Ladeluke which didn't even exist anymore.

So the missing files where already missing in the previous (broken) link. Now at least the link to Zenodo offers a working dataset repo. That's some progress in these places.

Reply:
ok, the link wasn't broken, the Ladeluke exists. But its content is woefully outdated, messy and doesn't contain the mentioned files as well.

Reply:
Zenodo also does not contain these files.
This change might give a wrong idea, that Zenodo is "broken".
Zenodo repo is up to date for its purpose, but not for holding random data for tutorial - which was more appropriate to remain in Ladeluke, from my pov.

Reply:
Then we should revert the Ladeluke links despite it's containing mostly quite old presentational files?

The new Zenodo link on the homepage still seems valid and useful though.

Reply:
Yes, i would revert to Ladeluke and have another task on Jira TVB to fix these tutorials and provide up to date data in Ladeluke.
Yes, the link on Zenodo on the home page, as well as replacing the deprecated tvb-data repo mention are perfect.

Reply:
Link changes on inner pages are now reverted to the old Ladeluke. 

The new homepage link to Zenodo is re-instated.

Will file a TVB Jira issue about the outdated content of the Ladeluke.